README.md

```diff
@@ -1,80 +1,154 @@
-twine check dist/* && twine upload --verbose dist/*.whl dist/*.zip
-```
-### Test release locally
-A simplified build and release workflow is available for testing locally.
-``` 
-python3 -m build --wheel && python3 setup.py sdist --format=zip
-pip3 install <product_name> --no-index --find-links file:///<path_to_repo>/dist/
-```
-## GitHub automation
-The [GitHub Action descriptor](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/workflows/python-publish.yml) runs through all the release products automatically when it detects a tag or release on the product repo.
\ No newline at end of file
+<!-- Header block for project -->
+<hr>
+
+<div align="center">
+
+[INSERT YOUR LOGO IMAGE HERE (IF APPLICABLE)]
+<!-- ☝️ Replace with your logo (if applicable) via ![](https://uri-to-your-logo-image) ☝️ -->
+<!-- ☝️ If you see logo rendering errors, make sure you're not using indentation, or try an HTML IMG tag -->
+
+<h1 align="center">[INSERT YOUR REPO / PROJ NAME HERE]</h1>
+<!-- ☝️ Replace with your repo name ☝️ -->
+
+</div>
+
+<pre align="center">[INSERT A SINGLE SENTENCE DESCRIBING THE PURPOSE OF YOUR REPO / PROJ]</pre>
+<!-- ☝️ Replace with a single sentence describing the purpose of your repo / proj ☝️ -->
+
+<!-- Header block for project -->
+
+[INSERT YOUR BADGES HERE (SEE: https://shields.io)] [![SLIM](https://img.shields.io/badge/Best%20Practices%20from-SLIM-blue)](https://nasa-ammos.github.io/slim/)
+<!-- ☝️ Add badges via: https://shields.io e.g. ![](https://img.shields.io/github/your_chosen_action/your_org/your_repo) ☝️ -->
+
+[INSERT SCREENSHOT OF YOUR SOFTWARE, IF APPLICABLE]
+<!-- ☝️ Screenshot of your software (if applicable) via ![](https://uri-to-your-screenshot) ☝️ -->
+
+[INSERT MORE DETAILED DESCRIPTION OF YOUR REPOSITORY HERE]
+<!-- ☝️ Replace with a more detailed description of your repository, including why it was made and whom its intended for.  ☝️ -->
+
+[INSERT LIST OF IMPORTANT PROJECT / REPO LINKS HERE]
+<!-- example links>
+[Website](INSERT WEBSITE LINK HERE) | [Docs/Wiki](INSERT DOCS/WIKI SITE LINK HERE) | [Discussion Board](INSERT DISCUSSION BOARD LINK HERE) | [Issue Tracker](INSERT ISSUE TRACKER LINK HERE)
+-->
+
+## Features
+
+* [INSERT LIST OF FEATURES IMPORTANT TO YOUR USERS HERE]
+  
+<!-- ☝️ Replace with a bullet-point list of your features ☝️ -->
+
+## Contents
+
+* [Quick Start](#quick-start)
+* [Changelog](#changelog)
+* [FAQ](#frequently-asked-questions-faq)
+* [Contributing Guide](#contributing)
+* [License](#license)
+* [Support](#support)
+
+## Quick Start
+
+This guide provides a quick way to get started with our project. Please see our [docs]([INSERT LINK TO DOCS SITE / WIKI HERE]) for a more comprehensive overview.
+
+### Requirements
+
+* [INSERT LIST OF REQUIREMENTS HERE]
+  
+<!-- ☝️ Replace with a numbered list of your requirements, including hardware if applicable ☝️ -->
+
+### Setup Instructions
+
+1. [INSERT STEP-BY-STEP SETUP INSTRUCTIONS HERE, WITH OPTIONAL SCREENSHOTS]
+   
```

Comment:
Here's a good spot to explain how to set up the:
1. Local dev env
2. GitHub Actions (if the dev person is the admin)